build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -95,4 +100,18 @@
     <!-- SetTelemetryProfile -->
     <SetEnvVar Name="DOTNET_CLI_TELEMETRY_PROFILE" Value="https://github.com/dotnet/cli;$(CommitHash)" />
   </Target>
+
+  <Target DependsOnTargets="Init" Name="RestorePackages">
+
+    <!-- Clean NuGet Temp Cache on Linux (seeing some issues on Linux) -->
+    <!-- TODO: do we still need this?  Seems unnecessary -->
+    <RemoveDir
+      Condition=" '$(OSName)' != 'win' AND '$(OSName)' != 'osx' "
+      Directories="/tmp/NuGet" />
+
+    <DotNetRestore WorkingDirectory="$(RepoRoot)/src" ToolPath="$(DotNetPath)" />
+    <DotNetRestore WorkingDirectory="$(RepoRoot)/tools" ToolPath="$(DotNetPath)" />
+
```

Comment:
Also, you can control your logging, like I did:

```C#
        protected override void LogToolCommand(string message)
        {
            base.LogToolCommand($"{GetWorkingDirectory()}> {message}");
        }
```

This allows us to see what directory we are restoring in the logs.